app/src/main/java/com/abisoft/todocompose/api/TodoApiService.kt

```diff
@@ -1,46 +1,53 @@
 package com.example.myapplication.okhttp_client
 
+import com.abisoft.todocompose.api.UpdateRequestData
+import com.abisoft.todocompose.model.TodoItemPost
 import retrofit2.Response
 import retrofit2.http.Body
 import retrofit2.http.DELETE
 import retrofit2.http.GET
 import retrofit2.http.Header
+import retrofit2.http.Headers
 import retrofit2.http.POST
 import retrofit2.http.PUT
 import retrofit2.http.Path
 
 interface TodoApiService {
 
-    @GET("list/")
-    suspend fun getTodoList(
-        @Header("Authorization") token: String,
+    @Headers("Authorization: Bearer Earendil")
```

Comment:
Note that in real app we should store any credentials like tokens, keys, password, etc in repository code. It should be provided different way (depends on case)